grid2op/Environment/MultiEnv2.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+from multiprocessing import Process, Pipe
+import numpy as np
+
+from grid2op.dtypes import dt_int
+from grid2op.Exceptions import Grid2OpException, MultiEnvException
+from grid2op.Space import GridObjects
+from grid2op.Environment import BaseMultiEnvironment
+from grid2op.Action import BaseAction
+
+# TODO test this class.
+
+class MultiEnvironment2(BaseMultiEnvironment):
+    """
+    This class allows to evaluate a single agent instance on multiple environments running in parrallel.
+
+    Attributes
+    -----------
+    envs: `list:grid2op.Environment.Environment`
+        Al list of environments for which the evaluation will be made in parallel.
+
+    nb_envs: ``list:int``
+        Number of parallel underlying environment that will be handled. 
```

Comment:
specify that the list should have the same size as the "envs" list and that total number of environments will be exactly the sum(nb_envs) maybe